Student.js

```diff
@@ -163,5 +163,18 @@ function Student(
         return this.fullName();
     };
 
+    this.checkPreferenceNumbers = function() {
```

Comment:
A more appropriate name for this function might be "checkAssigned" or "checkAssignedPreferences." This is a minor detail so it doesn't need to be changed right now, but it might be worth cleaning up if we have time in the future.

Side note: This function seems to share a lot of functionality with Student.calculateScore, maybe we should merge them into a single function at some point.

Reply:
While both functions compare the assigned workshops to the students preferences they return different things. Let me know if you find a solution